tesseract/server.py

```diff
@@ -161,6 +213,8 @@ def compile_select(self, result):
         # Extract the values for the expression.
         return (lua, args)
 
+    def publish(self, name, value):
```

Comment:
The standard tests override this, os its never actually run. Need a proper client test.